packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,138 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: c.$grid * 6;
+$w: c.$grid * 15;
+$labelMargin: c.$grid * 4;
+$trackBr: 22px;
+$trackBgOn: #1dbc22;
+$trackBgOff: c.$colorNeutralLight;
+$knobRight: $w * 0.6;
```

Comment:
Do we need this one at all? Could we `$knobOffset` as `right` or `left` instead depending on the state?

Reply:
great catch. fixed without magic constants.